apps/ima/index.js

```diff
@@ -33,9 +36,65 @@ module.exports = {
       behaviours: SaveFormSession,
       fields: ['who-are-you'],
       locals: { showSaveAndExit: true },
-      next: '/confirm', // TO BE UPDATED AS STEPS ARE ADDED
+      next: '/human-rights-claim', // TO BE UPDATED AS STEPS ARE ADDED
       backLink: 'current-progress'
     },
+    '/human-rights-claim': {
+      behaviours: SaveFormSession,
+      fields: ['human-rights-claim'],
+      locals: { showSaveAndExit: true },
+      forks: [
+        {
+          target: '/human-rights-family',
+          condition: req => req.sessionModel.get('human-rights-claim') === 'yes',
+          continueOnEdit: true
+        },
+        {
+          target: '/other-human-rights-claims',
+          condition: req => req.sessionModel.get('human-rights-claim') === 'no',
+          continueOnEdit: true
+        }
+      ],
+      continueOnEdit: true,
+      next: '/other-human-rights-claims'
+    },
+    '/human-rights-family': {
+      behaviours: SaveFormSession,
+      fields: ['family-member-relation', 'family-member-full-name', 'family-member-dob', 'family-member-nationality', 'uk-immigration-status', 'immigration-status-other', 'reference-number-option', 'uan-detail', 'ho-number-detail', 'human-rights-claim-details'],
```

Comment:
can you format this so it is each is on one line? Reads a bit long like this

Reply:
Separated them out into new lines.